src/profile.c

```diff
@@ -1512,8 +1512,77 @@ typedef struct profile_mpegts_spawn {
   char *pro_mime;
   int   pro_killsig;
   int   pro_killtimeout;
+  uint16_t pro_rewrite_sid;
+  int pro_rewrite_pmt;
+  int pro_rewrite_pat;
+  int pro_rewrite_sdt;
+  int pro_rewrite_nit;
+  int pro_rewrite_eit;
 } profile_mpegts_spawn_t;
 
+static int
+profile_spawn_rewrite_sid_set (void *in, const void *v)
+{
+  profile_mpegts_spawn_t *pro = (profile_mpegts_spawn_t *)in;
+  const uint16_t *val = v;
+  if (*val != pro->pro_rewrite_sid) {
+    if (*val > 0) {
+      pro->pro_rewrite_pmt =
```

Comment:
Please don't do this. Specify the value on each line, that makes it easier to read and less likely to break when changing this code later on.

Reply:
Understood! I also changed it at the position, where I copied it from. It is now fixed in the pass profile and in the spawn profile.